FirebaseAdmin/FirebaseAdmin/Messaging/FirebaseMessaging.cs

```diff
@@ -171,16 +171,175 @@ public async Task<string> SendAsync(Message message, bool dryRun)
                 message, dryRun, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Sends each message in the given list via Firebase Cloud Messaging. Unlike
+        /// <see cref="SendAllAsync(IEnumerable{Message})"/>, this method makes a single HTTP call
+        /// for each message in the given list.
+        /// </summary>
+        /// <exception cref="FirebaseMessagingException">If an error occurs while sending the
+        /// messages.</exception>
+        /// <param name="messages">Up to 500 messages to send in the batch. Cannot be null.</param>
```

Comment:
empty ok?

Reply:
I'm not sure what you mean. Could you clarify?

Reply:
non-null and non-empty? or just non null?

Reply:
I think empty is also not ok. It's the same for `SendAllAsync`, but we didn't explicitly mention this for `SendAllAsync` in the past. Should I change it to "Cannot be null or empty."  for both all `SendAll` APIs and all `SendEach` APIs? Or I can keep `SendAll` API as they are now and only update the comment for `SendEach` APIs?

Reply:
I kept `SendAll` API unchanged and updated to "Cannot be null or empty."  for all `SendEach` APIs.